packages/mroonga-package-task.rb

```diff
@@ -156,6 +156,11 @@ def original_archive_path
   end
 
   def define_archive_task
+    file original_archive_path.to_s do
+      source_archive_url = built_package_url(:source, @original_archive_name)
+      download(source_archive_url, "../..")
```

Comment:
Can we use `original_archive_path` here?

```suggestion
      download(source_archive_url, original_archive_path.to_s)
```

Reply:
fix: [616458e](https://github.com/mroonga/mroonga/pull/866/commits/616458e451da41f6ae616a3138b7deee94f3853b) Yes, we can use it. Thanks.